File: app/src/main/java/hut/cwp/mvppresenter/TestFragment.java
package hut.cwp.mvppresenter;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import hut.cwp.mylibrary.mvp.MvpFragment;

public class TestFragment extends MvpFragment<TestPresenter,ITest> {

    public TestFragment() {
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_test, container, false);
    }

}

File: sample/src/main/java/hut/cwp/mvppresenter/TestFragment.kt
package hut.cwp.mvppresenter

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import hut.cwp.annotations.mvp.DelegateBind
import hut.cwp.core.MvpFragment

@DelegateBind(presenter = TestPresenter::class)
class TestFragment : MvpFragment<TestPresenter, ITest >() {
    override fun onCreateView(
            inflater: LayoutInflater, container: ViewGroup,
            savedInstanceState: Bundle
    ): View {
        return inflater.inflate(R.layout.fragment_test, container, false)
    }
}